dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -138,7 +138,8 @@ public class ConnectorArguments extends DefaultArguments {
     private final OptionSpec<String> optionOracleSID = parser.accepts(OPT_ORACLE_SID, "SID name for oracle").withRequiredArg().describedAs("orcl").ofType(String.class);
     private final OptionSpec<String> optionConfiguration = parser.accepts("config", "Configuration for DB connector").withRequiredArg().ofType(String.class).withValuesSeparatedBy(';').describedAs("key=val;key1=val1");
     // private final OptionSpec<String> optionDatabase = parser.accepts("database", "database (can be repeated; all if not specified)").withRequiredArg().describedAs("my_dbname").withValuesSeparatedBy(',');
-    private final OptionSpec<File> optionOutput = parser.accepts("output", "Output file").withRequiredArg().ofType(File.class).describedAs("cw-dump.zip");
+    private final OptionSpec<File> optionOutput = parser.accepts("output", "Output file").withOptionalArg().ofType(File.class).describedAs("cw-dump.zip");
```

Comment:
1. I think this option should keep the `withRequiredArg` config, because it's the argument of the option that is required (`my-file.zip` in the expression `--output my-file.zip`), not the option itself.
2. I would suggest to change the description "Output file" to "Output file (cannot be used together with --output-dir)".